Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -109,12 +176,32 @@ public void NewHeight(long height)
             _ = _contexts[height].StartAsync(lastCommit);
         }
 
+        /// <summary>
+        /// Committing the block, which is voted by consensus, to the
+        /// <see cref="BlockChain{T}"/>.
```

Comment:
```suggestion
        /// Committing the block to the <see cref="BlockChain{T}"/>.
```
Description is out of context, duplication of <remarks> section.